packages/worker_bee/worker_bee_builder/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
   sdk: ^3.0.0
 
 dependencies:
-  analyzer: ^5.1.0
+  analyzer: ">=5.1.0 <7.0.0"
```

Comment:
@dnys1 Can the lower bound be set to 6.0.0?

Reply:
```suggestion
  analyzer: ">=6.0.0 <7.0.0"
```

Reply:
We need to support a at least down to 5.2 for built_value_generator. I am going to leave this at 5.1 for now.